x11-misc/fpm2/fpm2-0.90.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A GUI password manager utility with password generator"
+HOMEPAGE="https://als.regnet.cz/fpm2/"
+SRC_URI="https://als.regnet.cz/${PN}/download/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="2"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	dev-libs/glib:2
+	dev-libs/libxml2
+	dev-libs/nettle
+	x11-libs/gtk+:3
+	x11-libs/libX11"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-util/intltool
+	virtual/pkgconfig"
+
+src_prepare() {
+	default
+	local x
+	for x in data/${PN}.desktop.in; do
+		echo "${x}" >> po/POTFILES.in || die
+	done
```

Comment:
...this is a single item for loop?

Reply:
:smile: It had multiple items before, had to rework it a little bit, totally forgot to remove the loop afterwards. Fixed ;)